SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1816,6 +1819,11 @@ void USpatialNetDriver::TickFlush(float DeltaTime)
 	// Super::TickFlush() will not call ReplicateActors() because Spatial connections have InternalAck set to true.
 	// In our case, our Spatial actor interop is triggered through ReplicateActors() so we want to call it regardless.
 	Super::TickFlush(DeltaTime);
+
+	if (Connection != nullptr)
+	{
+		Connection->MaybeFlush();
```

Comment:
This call to MaybeFlush still checks the `bWorkerFlushAfterRPC` option, which seems incorrect

Reply:
It's intentional to flush the components but the logic here is slightly wrong, i'll add a comment. Perhaps the naming is incorrect!